lutris/sysoptions.py

```diff
@@ -227,6 +227,16 @@ def get_vk_icd_choices():
         "advanced": True,
         "help": _("Switch to US keyboard QWERTY layout while game is running"),
     },
+    {
+        "option": "nvidia_separate_glshaderdiskcache",
+        "type": "bool",
+        "default": True,
+        "label": _("Separate GL Shader Disk Caches (NVIDIA)"),
+        "advanced": True,
+        "help": _("Use per-game GL shader disk caches for NVidia hardware. "
+                  "This allows larger total caches. Turn this off for a single "
```

Comment:
"This allows to bypass the default shader cache size limitation in the NVIDIA driver and avoid unnecessary cache pruning for games with many shaders. Disable to use the default (shared) shader cache location."

Reply:
Sure thing. The real problem is naming the option "Koristite namjensko mjesto za predmemoriju shadera igre koju generira NVIDIA upravljački program".